app/core/migrations/0002_user_current_job_title_user_current_skills_and_more.py

```diff
@@ -0,0 +1,85 @@
+# Generated by Django 4.0.7 on 2022-09-20 23:06
+
+from django.db import migrations, models
+import phonenumber_field.modelfields
+import timezone_field.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='current_job_title',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='current_skills',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='first_name',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='github_handle',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='gmail',
+            field=models.EmailField(blank=True, max_length=254),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='last_name',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='linkedin_account',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='phone',
```

Comment:
Does this belong in the user model? I think we might want to model this as a Phone model (which has the ok_to_text field) with a ForeignKey to the user (and a unique_together = ['user', 'phone_number']

Reply:
I agree. That would be better.

Reply:
This is clearly the better design to me.
- Added #89 to make the change.